src/cps/pc_compat/meson.build

```diff
@@ -0,0 +1,34 @@
+_flex_version = '>= 2.6'
```

Comment:
@dcbaker Off topic, but the fact that I *can* ship boilerplate like this in a CMake modules is a killer feature for CMake at the scale I operate on for my day job. We literally have tens of thousands of codegen hooks across our codebase. Having a build system that allows central support for this sort of thing is non-optional at this point, even at the scale of a dozen repos with the same kind of code generation.

If I'm missing something and Meson does allow this sort of thing to be abstracted away into helper funcitons or something, maybe we should do that here, if only for the clearer code organization. Tips are helpful if you have any.

Reply:
Meson doesn't have functions or any Meson script abstractions, they all have to go upstream. This was an explicit design decision, caused by the lead developer's experience of working for a distro fixing build systems where everyone was cargo culting various cmake and m4 snippets around, where they came out of sync and were all broken differently.

So Meson makes the tradeoff in the other direction, abstractions go into Meson core. It means you don't get in code abstractions in your Meson.build flies, but it means that any abstractions are a) written in a real programming language, b) tested on several OSes before landing, c) get code review, and d) when there are bug fixes everyone gets those automatically.

I've been meaning to writing some abstractions for flex and bison in Meson, so I sat down and did that. Still have a few CI issues to work out: https://github.com/mesonbuild/meson/pull/13467